collector/cloudwatch/readme.md

```diff
@@ -0,0 +1,49 @@
+# cltr_cloudwatch
+
```

Comment:
This is small enough that I would inline this into the main plugin readme instead, we can split it later if and when we have a ton of collectors and it is too cluttered

Reply:
Done